Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -217,6 +221,14 @@ - (WMFArticleHeaderImageGalleryViewController*)headerGallery {
     return _headerGallery;
 }
 
+
+- (WMFTableOfContentsViewController*)tableOfContentsViewController{
+    if(!_tableOfContentsViewController){
+        _tableOfContentsViewController = [[WMFTableOfContentsViewController alloc] initWithSectionList:self.article.sections delegate:self];
```

Comment:
@bgerstle its cleaned up in the `setArticle:` method like all the other article dependent stuff.

Also been thinking about not caching this now though… and just recreating when needed.